packages/regenesis-surgery/test/uniswap.spec.ts

```diff
@@ -172,16 +210,16 @@ describe('uniswap contracts', () => {
 
       // TODO: add a test for minting positions?
     })
-
-    // Hack for dynamically generating tests based on async data.
-    // eslint-disable-next-line @typescript-eslint/no-empty-function
-    it('stub', async () => {})
   })
 
   describe('other', () => {
+    let accs
     before(async () => {
-      const accs = env.getAccountsByType(AccountType.UNISWAP_V3_OTHER)
+      accs = env.getAccountsByType(AccountType.UNISWAP_V3_OTHER)
+    })
 
+    // TODO: for some reason these tests fail
```

Comment:
Perhaps due to a config issue

Reply:
I think this needs to account for the update to the WETH address